libkbfs/backpressure_disk_limiter_test.go

```diff
@@ -433,6 +433,144 @@ func testBackpressureDiskLimiterLargeDiskDelay(
 	}, fileSnapshot)
 }
 
+// testBackpressureDiskLimiterLargeDiskDelay checks the delays when
+// pretending to have a large disk.
+func TestBackpressureDiskLimiterJournalAndDiskCache(
+	t *testing.T) {
+	t.Parallel()
+	var lastDelay time.Duration
+	delayFn := func(ctx context.Context, delay time.Duration) error {
+		lastDelay = delay
+		return nil
+	}
+
+	const blockBytes int64 = 100
+	// Big number, but no risk of overflow
+	maxFreeBytes := int64(1 << 30)
+
+	// Set the bottleneck; i.e. set parameters so that semaphoreMax for the
+	// bottleneck always has value 10 * blockX when called in beforeBlockPut,
+	// and every block put beyond the min threshold leads to an increase in
+	// timeout of 1 second up to the max.
+	byteLimit := 10 * blockBytes
+	// arbitrarily large number
+	fileLimit := int64(100000000)
```

Comment:
maybe `math.MaxInt64` would be a better constant to use if we don't care about it (and remove the *4 below)